tsfc/ufl_utils.py

```diff
@@ -21,6 +23,32 @@
                                      construct_modified_terminal)
 
 
+def compute_form_data(form,
```

Comment:
Exactly. I've spotted that you've changed them in the mean time so it scared me up.